src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -37,6 +40,14 @@ public partial class MainWindow
         const int ROW_HEIGHT = 75;
         const int MAX_LIST_HEIGHT = 300;
 
+        UISettings UISettings;
+        Color currentTheme;
+        Color LightTheme = Color.FromArgb(255, 255, 255, 255);
+        Color DefaultTheme = Color.FromArgb(255, 0, 0, 0);       
+        System.Windows.Media.SolidColorBrush BorderBrushLightTheme = new System.Windows.Media.SolidColorBrush(System.Windows.Media.Color.FromArgb(255, 255, 255, 255));

```

Comment:
Just curious, is this the only way to do it? Can't we set a theme resource for borders?

Reply:
I tried `Dynamic binding` with resource method but it doesn't reflect in the UI.